src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityTimestamp.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { MessageDescriptor } from 'react-intl';
+// @ts-ignore flow import
+import ActivityTimestamp from '../common/activity-timestamp';
+import AnnotationActivityLink from './AnnotationActivityLink';
+import './AnnotationActivityTimestamp.scss';
+
+type Props = {
+    date: number;
+    id: string;
+    isDisabled?: boolean;
+    message: MessageDescriptor;
+    onAnnotationSelect: () => void;
+};
+
+const AnnotationActivityTimestamp = ({
+    date,
+    id,
+    isDisabled = false,
+    message,
+    onAnnotationSelect,
+    ...rest
+}: Props): JSX.Element => (
+    <div className="bcs-AnnotationActivityTimestamp" {...rest}>
```

Comment:
Does this need to be its own component? It doesn't seem like its children share any commonalities and the naming doesn't imply it contains a link.